src/NServiceBus.Core/Logging/ColoredConsoleLogger.cs

```diff
@@ -1,26 +1,23 @@
 ﻿namespace NServiceBus
 {
     using System;
-    using System.Runtime.InteropServices;
+    using System.IO;
     using Logging;
 
     static class ColoredConsoleLogger
     {
         static ColoredConsoleLogger()
         {
-            const int STD_OUTPUT_HANDLE = -11;
-            logToConsole = GetStdHandle(STD_OUTPUT_HANDLE) != IntPtr.Zero;
+            logToConsole = Console.OpenStandardOutput() != Stream.Null;
```

Comment:
Overall, I'm thinking it's probably fine to not dispose it based on what I'm seeing, but to play it safe, I'll go ahead and wrap the call in a using statement.